addons/core/functions/server/fnc_serverInit.sqf

```diff
@@ -18,95 +18,129 @@
+            WARNING('Deprecated variable used: tf_freq_guer (new: TFAR_defaultFrequencies_sr_independent) in ADDON'); \
+            TFAR_defaultFrequencies_sr_independent = tf_freq_guer param [2,nil];
+        };
+        if (!isNil "tf_freq_guer_lr") then { \
+            WARNING('Deprecated variable used: tf_freq_guer_lr (new: TFAR_defaultFrequencies_lr_independent) in ADDON'); \
+            TFAR_defaultFrequencies_lr_independent = tf_freq_guer_lr param [2,nil];
+        };
+
+        if (TFAR_SameSRFrequenciesForSide) then {
+            TFAR_freq_sr_west = false call DFUNC(generateSRSettings);
+            TFAR_freq_sr_west set [2,TFAR_defaultFrequencies_sr_west];
+            TFAR_freq_sr_east = false call DFUNC(generateSRSettings);
+            TFAR_freq_sr_east set [2,TFAR_defaultFrequencies_sr_east];
+            TFAR_freq_sr_independent = false call DFUNC(generateSRSettings);
+            TFAR_freq_sr_independent set [2,TFAR_defaultFrequencies_sr_independent];
+            publicVariable "TFAR_freq_sr_west";
+            publicVariable "TFAR_freq_sr_east";
+            publicVariable "TFAR_freq_sr_independent";
+        } else {
+            VARIABLE_DEFAULT(TFAR_freq_sr_west,true call DFUNC(generateSRSettings));
+            VARIABLE_DEFAULT(TFAR_freq_sr_east,true call DFUNC(generateSRSettings));
+            VARIABLE_DEFAULT(TFAR_freq_sr_independent,true call DFUNC(generateSRSettings));
+        };
+
+        if (TFAR_SameLRFrequenciesForSide) then {
+            TFAR_freq_lr_west = false call DFUNC(generateLrSettings);
+            TFAR_freq_lr_west set [2,TFAR_defaultFrequencies_lr_west];
+            TFAR_freq_lr_east = false call DFUNC(generateLrSettings);
+            TFAR_freq_lr_east set [2,TFAR_defaultFrequencies_lr_east];
+            TFAR_freq_lr_independent = false call DFUNC(generateLrSettings);
+            TFAR_freq_lr_independent set [2,TFAR_defaultFrequencies_lr_independent];
+            publicVariable "TFAR_freq_lr_west";
+            publicVariable "TFAR_freq_lr_east";
+            publicVariable "TFAR_freq_lr_independent";
+        } else {
+            VARIABLE_DEFAULT(TFAR_freq_lr_west,true call DFUNC(generateLrSettings));
+            VARIABLE_DEFAULT(TFAR_freq_lr_east,true call DFUNC(generateLrSettings));
+            VARIABLE_DEFAULT(TFAR_freq_lr_independent,true call DFUNC(generateLrSettings));
+        };
+
+        //Check if all players are running TFAR
+        {
+            if(isServer) exitWith {};
+            waitUntil {sleep 0.1;time > 3};
+            if !(isClass(configFile >> "CfgPatches" >> "tfar_core")) exitWith {
+                [player, localize LSTRING(WM_NotLoad)] remoteExec ["globalChat", -2];
+                if (isNull (uiNamespace getVariable ["BIS_fnc_arsenal_cam", objNull])) then {
+                    [localize LSTRING(WM_NotLoad_Desc)] call "BIS_fnc_guiMessage";
+                };
+            };
+        } remoteExec ["BIS_fnc_spawn", -2, true];
```

Comment:
just remoteExec `spawn` instead.

Reply:
Actually this should really be a clientside function that can be whitelisted in CfgRemoteExec.. But don't wanna do this in this PR.

Reply:
actually, we can't create a clientside function, because this is the message to be displayed, if you join a server running tfar.
But i have to remove the localizing, because it wouldn't work.

Reply:
Duh... :D 
Seems like multiple people didn't think here then :D
You can pass the localized strings as parameter via remoteExec. But they will be localized to the server not to the client. And I think servers are always english...

Reply:
Yes, that's why we can leave the localizing out and put the string in as text.